ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClusterDetailsPage.tsx

```diff
@@ -109,6 +129,13 @@ function ClusterDetailsPage() {
                     route={coverageClusterDetailsPath.replace(':clusterId', clusterId)}
                 />
             </PageSection>
+            <PageSection>
```

Comment:
In the [mock](https://www.figma.com/proto/sbH8mKbTNSqgj3lm5CK6xS/Compliance-2.0?node-id=3706-334075&t=ecQTHg3xSDgzuCKP-0&scaling=scale-down&page-id=2826%3A400903), I noticed that they display this second header for the profile name, some subtext, and label chips to the right. Is that something we will also add?

Reply:
I didn't really touch on this, but the api used for profile information and benchmarks associations is still being worked on. I'm pushing for that endpoint to include information you're referring to.